### PR TITLE
Update to latest quasar & test-starter

### PR DESCRIPTION
**Pull request by garyb**
Also use the get-quasar script from `quasar-test-starter` rather than maintaining a copy in here.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "private": true,
   "scripts": {
-    "postinstall": "node script/get-quasar.js",
+    "update-quasar": "node bower_components/purescript-quasar-test-starter/quasar/get-quasar.js",
```
Comment (kritzcreek) [inline package.json:4]: yuck

**Review by kritzcreek — APPROVED** [review]
If you tested it works I'm 👍

File: package.json (hunk 1 above)
Comment (garyb) [reply package.json:4]: Yeah, but it's better than maintaining the script in multiple places and having to update it every time we change something 😕

Comment (kritzcreek) [reply package.json:4]: Hopefully my Rust script will be a better solution ^^

**Pull request merged**